content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,301 +1,217 @@
 
-In order for the Serverless Framework to provision the AWS resources and deploy your application, you must configure your access key.
-
-The method you use to get the access key depends on whether you use your personal account or create a special IAM user for the Serverless Framework. We recommend that you create a special IAM user because it lets you set granular permissions. To get an access key for your account, follow the [AWS guide](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey)
-
-When you have the _Access key ID_ and a _Secret access key_, set them with the following command. In the following command, replace the placeholders with your _Access key ID_ and _Secret access key_.
+## Deploy only the required files
 
-Use the following command:
+To reduce your deployment footprint, you can change your deployment process so that you only upload the files that your application needs. The Serverless configuration file `serverless.yml` below show a `package` pattern that includes only the Prisma engine file relevant to the Lambda runtime, and excludes the others. This means that when Serverless Framework packages your app for upload, it includes only one engine file. This ensures the packaged archive is as small as possible.
 
-```terminal wrap
-serverless config credentials --provider aws --key AWS_ACCESS_KEY_ID  --secret AWS_SECRET_ACCESS_KEY
+```code file=serverless.yml no-copy
+package:
+  patterns:
+    - '!node_modules/.prisma/client/libquery_engine-*'
+    - 'node_modules/.prisma/client/libquery_engine-rhel-*'
+    - '!node_modules/prisma/libquery_engine-*'
+    - '!node_modules/@prisma/engines/**'
 ```
 
-### Result
+However, if you use `serverless-webpack`, then you cannot use this method. See [Deployment with serverless webpack](#deployment-with-serverless-webpack) below.
 
-If the configuration is successful, you see the following result in the terminal.
+## Deployment with <inlinecode>serverless-webpack</inlinecode>
 
-```terminal
-✔ Profile "default" has been configured
-```
+If you use `serverless-webpack`, then you will need additional configuration so that your `schema.prisma` is properly bundled. You will need to:
 
-## 5. Deploy your project
+1. Copy your `schema.prisma` with [`copy-webpack-plugin`](https://www.npmjs.com/package/copy-webpack-plugin).
+2. Run `prisma generate` via `custom > webpack > packagerOptions > scripts` in your `serverless.yml`
+3. Only package the correct Prisma engine file. This can save more than 40mb of capacity.
 
-You have finished the configuration, so you can now deploy your project with `serverless deploy`.
+Let's get started!
 
-### Steps
+### Install webpack specific dependencies
```

Comment:
I would suggest using a numbered list for the following sections under deployment with `serverless-webpack`. 
```suggestion
### Install webpack specific dependencies
```

Reply:
done! thanks